src/main/java/org/broadinstitute/hellbender/utils/QualityUtils.java

```diff
@@ -5,6 +5,9 @@
 /**
  * QualityUtils is a static class with some utility methods for manipulating
  * quality scores.
+ *
+ * The definition of a Phred-scaled quality Q in terms of an error probability e is:
```

Comment:
`e` is not a great symbol to use when logs are nearby. Use another symbol in this comment